src/_includes/macros/active-filters.html

```diff
@@ -0,0 +1,143 @@
+
+{# ==========================================================================
+
+   active_filters()
+
+   Description:
+
+   Create active filters markup when given:
+
+   filters:  An array of the filters that you want.
+             Possible values: activity_category, author, calendar, category,
+             range_date, tags.
+
+   query:    The post type query object like `queries.posts`.
+             Needed so we can utilize `query.possible_values_for()` to get access to
+             the values of each filter.
+
+   posts:    The query object search results.
+             Used to show how many filtered items resulted.
+
+   doc_type: The type of document that posts contains.
+             Mainly used for conditionals needed to support newsroom filters.
+
+   options:  An object of non-required options that can be overriden.
+
+   options.expand_label:          The label you want on the expandable button.
+
+   options.show_unfiltered_count: Allows you to enable the count
+                                  label when no filters are applied.
+
+   options.show_filters_label:    Allows you to disable showing the count label.
+
+   options.use_list :             Boolean indicating wether to use list markup.
+
+   ========================================================================== #}
+
+{% macro render(posts, filters, options) %}
+
+{% set active_filters = [] %}
+{% set active_date_filters = [] %}
+{% set use_list = options.use_list or false %}
+
+{# This is a hack. `_ignore` is a variable we will never use but we can use
```

Comment:
We could use `do` syntax if the [expression-statement extension](http://jinja.pocoo.org/docs/dev/templates/#expression-statement) were loaded. Is this something that could happen easily with the transition to django? cc @dpford